tensorrt_llm/executor/proxy.py

```
@@ -617,6 +617,22 @@ def get_stats(self, timeout: float) -> List[dict]:
+        """Get static KV cache capacity from the runtime via RPC."""
+        if self.rpc_client is None:
+            logger.warning(
+                "RPC client not initialized, cannot get kv cache capacity")
+            return []
+
+        try:
+            capacity = self.rpc_client.fetch_kv_cache_capacity_async().remote()
+            return [
+                json.loads(c) if isinstance(c, str) else c for c in capacity
+            ]
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)